test/fixtures/payload.json

```diff
@@ -9,7 +9,7 @@
       "merged": {
         "data": {
           "title": "Pull Request with unreviewed and merged",
-          "labels": [{ "name": "PR: unreviewed" }],
+          "labels": [{ "name": "Status: Review Needed :passport_control:" }],
```

Comment:
Label `Status: Review Needed :passport_control:` ?

Reply:
I needed to have consistent labels with my set but it's been reverted.

Reply:
@TiagoDanin Thank you for your help. I would appreciate if you also review this PR.

Reply:
@Berkmann18 But this is part of another test: "PR: merged"
"PR: merged" Test:
- Load playload "merged"
- Event received: "merged"
- Check Labels
- Remove labels of PR-triage ("PR: unreviewed" Not found... and label "Status: Review Needed :passport_control:" is not part of PR-triage)
- Add label "PR: merged"
- Done!

Reply:
@TiagoDanin Again, I reverted that, look at the latest commit.